Sources/XcodeGenKit/SourceGenerator.swift

```diff
@@ -391,6 +391,8 @@ class SourceGenerator {
 
         let children = try getSourceChildren(targetSource: targetSource, dirPath: path, excludePaths: excludePaths, includePaths: includePaths)
 
+        let createIntermediateGroups = targetSource.createIntermediateGroups ?? project.options.createIntermediateGroups
```

Comment:
Now that this property is created, can we remove the parameter, or is that still required?
On the other hand, on hindsight I'm ok with just using the parameter, so the fallback logic is in a single place

Reply:
I preferred removing the parameter altogether and get the data from where it comes the target source itself or the project.